src/classes/BDI_DataImportService.cls

```diff
@@ -78,10 +78,14 @@ global with sharing class BDI_DataImportService {
     * @return DataImportBatch__c
     */
     private static DataImportBatch__c loadBatch(Id batchId) {
-
         List<DataImportBatch__c> dataImportBatches = BDI_DataImportBatch_SEL.selectByIds(new Set<Id>{ batchId });
         if (dataImportBatches.size() == 1) {
-            return dataImportBatches[0];
+            DataImportBatch__c currentBatch = dataImportBatches[0];
+            if (!String.isBlank(currentBatch.Active_Fields__c)) {
+                relevantDataImportFields = BGE_BatchGiftEntry_UTIL.getDataImportFields(batchId, false);
+                batchFromBGE = true;
```

Comment:
Want to avoid references to "BGE" if we can describe this property more generically ... "batchDefinesActiveFieldList" or somethign